server.go

```diff
@@ -396,6 +396,12 @@ func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
 		s.htlcSwitch.CloseLink(chanPoint, closureType, 0)
 	}
 
+	// We will use the following two channels to reliably hand off contract
+	// breach and close events from the ChannelArbitrator to the
+	// breachArbiter,
+	contractBreaches := make(chan *ContractBreachEvent, 1)
```

Comment:
We might consider making this unbuffered, just so that a breach event can't be abandoned in the channel when exiting

Reply:
I don't think this will be a problem, since we will wait for an ACK regardless.